pinot-segment-local/src/test/java/org/apache/pinot/segment/local/recordtransformer/ComplexTypeTransformerTest.java

```diff
@@ -237,4 +238,76 @@ public void testUnnestCollection() {
     next = itr.next();
     Assert.assertEquals("v2", next.getValue("array.a"));
   }
+
+  @Test
+  public void testConvertCollectionToString() {
```

Comment:
Can we please add a test where in a JSON object, one collection is unnested and another parallel collection at the same level in the root is not unnested ? I am assuming the one that is not unnested will still be repeated as many times based on the repetition level of the unnested collection.

Reply:
Yes. This is covered by the test `testUnnestCollection ` above